x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -183,6 +184,8 @@ export interface GetAlertInstanceSummaryParams {
   dateStart?: string;
 }
 
+const extractedSavedObjectParamReferenceNamePrefix = 'param:';
```

Comment:
**optional nit:** should we add a comment about modifying this value will require a migration for existing SOs with references?